ui/app/templates/components/alphabet-edit.hbs

```diff
@@ -116,7 +116,6 @@
           @value={{get this.model attr.name}}
           @type={{attr.type}}
           @isLink={{eq attr.name "transformations"}}
-          @viewAll="transformations"
```

Comment:
tested this throughout and using `@label` instead for this component works as expected!